.github/workflows/ci_dependabot_sync.yml

```
@@ -0,0 +1,38 @@
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version:  '3.4.7
+          bundler-cache: false
+
+      - name: Update decidim-generators Gemfile.lock if root changed
+        if: contains(github. event.pull_request.files, 'Gemfile.lock')
+        run: |
+          cd decidim-generators
+          bundle install
+
+      - name:  Commit changes
```

YAMLlint (1.38.0)

[warning] too many spaces after colon

(colons)